Source/lua/randomat2/events/gravity.lua

```diff
@@ -0,0 +1,77 @@
+local EVENT = {}
+
+CreateConVar("randomat_gravity_timer", 30, {FCVAR_ARCHIVE, FCVAR_NOTIFY}, "Timer (def. 30)", 5,90)
+CreateConVar("randomat_gravity_duration", 3, {FCVAR_ARCHIVE, FCVAR_NOTIFY}, "Duration (def. 3)", 1,15)
+CreateConVar("randomat_gravity_minimum", 70, {FCVAR_ARCHIVE, FCVAR_NOTIFY}, "The minimum gravity (def. 70)",10,500)
+CreateConVar("randomat_gravity_maximum", 2000, {FCVAR_ARCHIVE, FCVAR_NOTIFY}, "The minimum gravity (def. 70)",700,4000)
+
+EVENT.Title = "I don't think you realise the gravity of the situation."
+EVENT.id = "gravity"
+EVENT.AltTitle = "Gravity Changer"
+
+local defaultGravity = 600
+
+local function SetGravity(increase)
+    if increase then
+        RunConsoleCommand("sv_gravity", GetConVar("randomat_gravity_maximum"):GetFloat())
```

Comment:
There's probably some discussion to be had about using `sv_gravity` rather than Player:SetGravity. Doing it this way does mean that players who join late or are resurrected after the event starts get the gravity effect (as opposed to the Moon Gravity event where that is not the case)

Reply:
Actually it looks like Moon Gravity doesn't have a problem because it runs on a timer and re-sets players individual gravity repeatedly.

Reply:
That way people can just change the cvar to the exact values they want for gravity, no math needed too.

Reply:
Only downside I can think of is it prints to chat each time it changes.
Speaking of which -- do we want to add a message when the gravity flips?

Reply:
Hmm it prints it already so it would probably be overkill

Reply:
Yea, fair enough.